### ISSUE TITLE
Error raised with SpaceAfterPropertyColon reporting if missing semi-colon in property declaration

### ISSUE THREAD

**Issue by sjdemartini** [brigade/scss-lint#139]
Running a linter on an SCSS file with the SpaceAfterPropertyColon linter enabled causes this error for me:

```
SCSSLint::Linter::SpaceAfterPropertyColon raised unexpected error linting file forms.scss: 'Length must be more than 0, passed -2'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/location.rb:14:in `initialize'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter.rb:51:in `new'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter.rb:51:in `location_from_range'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter.rb:27:in `add_lint'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter/space_after_property_colon.rb:18:in `visit_prop'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/sass-3.3.5/lib/sass/tree/visitors/base.rb:36:in `visit'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter.rb:120:in `visit'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter.rb:130:in `block in visit_children'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter.rb:128:in `each'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter.rb:128:in `visit_children'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/sass-3.3.5/lib/sass/tree/visitors/base.rb:38:in `visit'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter.rb:120:in `visit'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter.rb:130:in `block in visit_children'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter.rb:128:in `each'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter.rb:128:in `visit_children'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/sass-3.3.5/lib/sass/tree/visitors/base.rb:38:in `visit'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter.rb:120:in `visit'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter.rb:130:in `block in visit_children'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter.rb:128:in `each'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter.rb:128:in `visit_children'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/sass-3.3.5/lib/sass/tree/visitors/base.rb:38:in `visit'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter.rb:120:in `visit'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/linter.rb:18:in `run'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/runner.rb:43:in `block in find_lints'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/runner.rb:38:in `each'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/runner.rb:38:in `find_lints'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/runner.rb:22:in `block in run'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/runner.rb:21:in `each'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/runner.rb:21:in `run'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/lib/scss_lint/cli.rb:94:in `run'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/bin/scss-lint:7:in `block in <top (required)>'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/bin/scss-lint:5:in `tap'
/Users/sjd/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.23.0/bin/scss-lint:5:in `<top (required)>'
/Users/sjd/.rvm/gems/ruby-2.1.1/bin/scss-lint:23:in `load'
/Users/sjd/.rvm/gems/ruby-2.1.1/bin/scss-lint:23:in `<main>'
/Users/sjd/.rvm/gems/ruby-2.1.1/bin/ruby_executable_hooks:15:in `eval'
/Users/sjd/.rvm/gems/ruby-2.1.1/bin/ruby_executable_hooks:15:in `<main>'
```

The source of the exception was the following SCSS:

```
span {
  color:#eee
}
```

missing the semicolon at the end of the property line, as well as having no space after the semi-colon. I am able to reproduce the exception consistently with the above situation, but I am unfamiliar with the root cause.
